Integrations/integration-Office_EWS.yml

```diff
@@ -1404,5 +1443,24 @@ script:
     - contextPath: EWS.Items.Headers.#text
       description: Mail attachment text
     description: Get actual attachment item from email
+  - name: ews-move-items
+    arguments:
+    - name: target-mailbox
+      description: The user email address
+    - name: item-ids
+      required: true
+      description: Comma separated list of Item IDs (result of the ews-search-mailbox
+        command)
+    - name: folder-identifier
+      required: true
+      description: The folder id
```

Comment:
Please give example also. `(E.g. 'inbox', 'sentitems'). All folder-ids are listed in ews-find-folders command`